chain/shard_tracker/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
I don't think this is really a chain thing... 
I also find it a bit weird as a separate Cargo.
I would just add it as a file to near/